tidb-read-staleness.md

```diff
@@ -0,0 +1,167 @@
+
+After enabling `tidb_read_staleness`, you still can perform the following operations:
+
+- Insert, modify, delete data or perform DML operations in the current session. These statements are not affected by `tidb_read_staleness`.
+- Start an interactive transaction in the current session. Queries within this transaction still read the latest data.
+
+After reading the historical data, you can read the latest data in the following two ways:
+
+- Start a new session.
+- Set the value of the `tidb_read_staleness` variable to `""` using the `SET` statement.
+
+## Usage examples
+
+This section describes how to use `tidb_read_staleness` with examples.
+
+1. Create a table, and insert a few rows of data into the table:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    create table t (c int);
+    ```
+
+    ```
+    Query OK, 0 rows affected (0.01 sec)
+    ```
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    insert into t values (1), (2), (3);
+    ```
+
+    ```
+    Query OK, 3 rows affected (0.00 sec)
+    ```
+
+2. Check out the data in the table:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select * from t;
+    ```
+
+    ```
+    +------+
+    | c    |
+    +------+
+    |    1 |
+    |    2 |
+    |    3 |
+    +------+
+    3 rows in set (0.00 sec)
+    ```
+
+3. Update the data in a row:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    update t set c=22 where c=2;
+    ```
+
+    ```
+    Query OK, 1 row affected (0.00 sec)
+    ```
+
+4. Confirm that the data has been updated:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select * from t;
+    ```
+
+    ```
+    +------+
+    | c    |
+    +------+
+    |    1 |
+    |   22 |
+    |    3 |
+    +------+
+    3 rows in set (0.00 sec)
+    ```
+
+5. Set the `tidb_read_staleness` system variable.
+
+    The scope of this variable is `SESSION`. After setting its value, TiDB reads the latest version data before the time set by the value.
+
+    The following setting indicates that TiDB selects a timestamp as new as possible within the time range from 5 seconds ago to now and uses it as the timestamp for reading historical data:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    set @@tidb_read_staleness="-5";
+    ```
+
+    ```
+    Query OK, 0 rows affected (0.00 sec)
+    ```
+
+    > **Note:**
+    >
+    > Use `@@` instead of `@` before `tidb_read_staleness`. `@@` means system variables, and `@` means user variables.
+    > You need to set the historical time range (the value of `tidb_read_staleness`) according to the total time that you spent in step 3 and step 4. Otherwise, the latest data will be displayed in the query results, not the historical data. Therefore, you need to adjust this time range according to the time you spent on operations. For example, in this example, since the set time range is 5 seconds, you need to complete step 3 and step 4 within 5 seconds.
```

Comment:
```suggestion
    >  - Use `@@` instead of `@` before `tidb_read_staleness`. `@@` means system variables, and `@` means user variables.
    >  - You need to set the historical time range (the value of `tidb_read_staleness`) according to the total time that you spent in step 3 and step 4. Otherwise, the latest data will be displayed in the query results, not the historical data. Therefore, you need to adjust this time range according to the time you spent on operations. For example, in this example, since the set time range is 5 seconds, you need to complete step 3 and step 4 within 5 seconds.
```